src/cmakeProject.ts

```diff
@@ -2874,6 +2874,22 @@ export class CMakeProject {
         return env;
     }
 
+    async cacheVariable(name?: string, defaultValue?: string): Promise<string | null> {
+        if (await this.needsReconfigure()) {
+            await this.configureInternal(ConfigureTrigger.api, [], ConfigureType.Normal);
+        }
+        if (!name) {
+            void vscode.window.showErrorMessage(localize('cachevariable.missing.required.argument.name', 'cacheVariable: missing required argument "name".'));
+            return null;
+        }
+        const value = (await CMakeCache.fromPath(await this.cachePath))?.get(name)?.value ?? defaultValue;
+        if (value === null || value === undefined) {
+            void vscode.window.showErrorMessage(localize('cachevariable.not.found', `Variable "${name}" not found in CMake cache, and no default provided.`));
```

Comment:
So for the localize method, instead of using string interpolation, you should use something like this

```suggestion
            void vscode.window.showErrorMessage(localize('cachevariable.not.found', `Variable "{0}" not found in CMake cache, and no default provided.`, name));
```

Reply:
Gotcha. I found a bunch of other examples in `cmakeProjects.ts` of localize being called like that, but with the quotes in the third argument like `` `"${name}"` ``. I followed that pattern and pushed the fix before seeing your comment. Let me know if you want me to swap it out for your exact suggested change.

Reply:
I agree with you on that nit, I was just playing "when in Rome" doing it the other way. Happy to revise.